test/k8sT/Tunnels.go

```diff
@@ -30,12 +30,13 @@ var _ = Describe("K8sTunnelTest", func() {
 	var kubectl *helpers.Kubectl
 	var demoDSPath string
 	var initialized bool
+	var logger *logrus.Entry
 
 	initialize := func() {
 		if initialized == true {
 			return
 		}
-		logger := log.WithFields(logrus.Fields{"testName": "K8sTunnelTest"})
+		logger := logrus.WithFields(logrus.Fields{"testName": "K8sTunnelTest"})
```

Comment:
There is a package scoped `log` object (that is a `*logrus.Logger` type) that you can use. If you use the package level function you get the default logrus logger as the base, and this is the one that receives all the library debug messages we were trying to exclude. Also, I think the `:=` shadows the more global variable which you added here.
tl;dr `logger = log.WithFields(logrus.Fields{"testName": "K8sTunnelTest"})` (note I dropped the `:=`